administration/src/cards/hashCardInfo.test.ts

```diff
@@ -183,8 +183,8 @@ describe('hashCardInfo', () => {
         extensionBirthday: new BirthdayExtension({
           birthday: -365 * 10,
```

Comment:
We should also add another test that sets the new identifier to passid. (Also in the frontend)